Update documentation (#221)
OpenLiberty/liberty-tools-intellij: docs/user-guide.md

### Gradle built application

To view the test report for Gradle built applications, you can either right-click on the application listed in the Liberty tool window, and click on the `View test report` action or select the `Liberty: View test report` action in the "Go to Action" IntelliJ IDEA menu.

Note that this action will look for the test report at the default location `build/reports/tests/test/index.html`.

## Stopping your application

To stop your application, you can either right-click on the application listed in the Liberty tool window, and click on the `Stop` action or select the `Liberty: Stop` action in the "Go to Action" IntelliJ IDEA menu.

![Liberty stop action](images/liberty-stop-action.png)

## Debugging your application

To debug your application, you can start dev mode with the debugger automatically attached to the Liberty server JVM running your application.

To start dev mode with the debugger attached, first create or select a Liberty Run/Debug Configuration through the IntelliJ Run/Debug Configuration menu. Once you select the Liberty Run/Debug Configuration, select the `Debug` action next to the Run/Debug Configuration menu.

> 

Author: Yes it's the name of the UI window - "Run/Debug Configurations"
<img width="1051" alt="image" src="https://user-images.githubusercontent.com/26146482/207717387-a0874d35-2713-4260-8913-ffb3723352ff.png">

> 

Reviewer: Ok- I didn't see that window title in any of the screenshots in the doc. But if that's what the user sees then ok

To manually add your Liberty project to the Liberty tool window, select the `Liberty: Add project to the tool window` action in the "Go to Action" IntelliJ IDEA menu. 

![Liberty add project to tool window](images/liberty-add-project-to-tool-window.png)

You will be prompted with a list of projects that are not already displayed in the Liberty tool window.

![Liberty add project to tool window selection](images/liberty-add-project-to-tool-window-selection.png)

You can remove manually added Liberty projects from the Liberty tool window by selecting the `Liberty: Remove project from the tool window` action in the "Go to Action" IntelliJ IDEA menu.

## Configuring a Liberty server

Liberty configuration assistance is offered through the Liberty Config Language Server. For more information, see the [project documentation in GitHub](https://github.com/OpenLiberty/liberty-language-server#liberty-config-language-server).

1. Start the project in dev mode, using one of the Liberty tool window start commands above. This will install the Liberty features required for your application and allow generation of a corresponding server.xml schema file.

> 2. Open your server.xml file. Proceed to use Liberty specific editing support: completion by typing "Ctl/Cmd + Space" at a given point within the document.

Reviewer: ```suggestion
2. Open any of the supported Liberty configuration files. 
```

> 3. Open your bootstrap.properties or server.env file. Proceed to use Liberty specific editing support: completion by typing "Ctl/Cmd + Space" at a given point within the document.

Reviewer: ```suggestion
3. To use Liberty-specific code completion, press **Ctl/Cmd** + **Space** anywhere within the document. A drop-down list of completion suggestions appears.
```

![LCLS server.xml completion](images/LCLS-server-xml-completion.png)

> ## Developing with Jakarta EE and MicroProfile APIs

Reviewer: ```suggestion
## Develop with Jakarta EE and MicroProfile APIs
```

> 

Reviewer: suggest revising this section in a similar fashion to the previous section suggestions
